docs/octant-plugin-installation.md

```diff
@@ -75,41 +75,54 @@ You can follow the steps listed below to install octant and antrea-octant-plugin
 1. Get and install Octant v0.13.1.
 
     Depending on your linux operating system, to install Octant v0.13.1, you can use either
-    ```
+    ```bash
     wget https://github.com/vmware-tanzu/octant/releases/download/v0.13.1/octant_0.13.1_Linux-64bit.deb
     dpkg -i octant_0.13.1_Linux-64bit.deb
     ```
     or
-    ```
+    ```bash
     wget https://github.com/vmware-tanzu/octant/releases/download/v0.13.1/octant_0.13.1_Linux-64bit.rpm
     rpm -i octant_0.13.1_Linux-64bit.rpm
     ```
 
 2. Export your kubeconfig path (file location depends on your setup) to environment variable $KUBECONFIG.
 
-    ```
+    ```bash
     export KUBECONFIG=/etc/kubernetes/admin.conf
     ```
 
-3. Build antrea-octant-plugin.
+3. Get corresponding antrea-octant-plugin binary from [Release Assets](https://github.com/vmware-tanzu/antrea/releases/tag/v0.8.1)
```

Comment:
I feel it should link to https://github.com/vmware-tanzu/antrea/releases and not a specific release, otherwise you will need to update it whenever a new release.

Reply:
Done